tst/RtcpFunctionalityTest.cpp

```diff
@@ -133,6 +133,38 @@ TEST_F(RtcpFunctionalityTest, onRtcpPacketCompound) {
     EXPECT_EQ(STATUS_SUCCESS, onRtcpPacket(&peerConnection, compound, SIZEOF(compound)));
 }
 
+TEST_F(RtcpFunctionalityTest, onRtcpPacketCompoundSenderReport)
```

Comment:
Looks great! Can we also add a unit test to add multiple transceivers and pass one of them `getRtpRemoteInboundStats` arg?